File-Formats/PDF/Convert-HTML-To-PDF/blazor.md

```diff
@@ -11,16 +11,16 @@ keywords: Assemblies
 
 Steps to convert HTML to PDF in Blazor application
 
-Create a new C# Blazor application project. Select Blazor App from the template and click the Next button.
+1. Create a new C# Blazor application project. Select Blazor App from the template and click the Next button.
```

Comment:
Create a new C# Blazor Server application project.

Reply:
modified the code changes